guides/getting-started.md

```diff
@@ -1,45 +1,42 @@
-Get started with Angular Material using the Angular CLI.
+For help getting started with a new Angular app, check out the [Angular CLI](https://cli.angular.io/).
 
-## Install Angular CLI
- 
- ```bash
- npm install -g @angular/cli
- ```
- 
-## Create a new project
- 
- ```bash
- ng new my-project
- ```
-
-The new command creates a project with a build system for your Angular app.
+For existing apps, follow these steps to begin using Angular Material.
 
-## Install Angular Material components 
+## Step 1: Install Angular Material 
 
 ```bash
 npm install --save @angular/material
 ```
 
-## Import the Angular Material NgModule
+## Step 2: Import the Module
+  
+Add MaterialModule as an import in your app's root NgModule.  
   
-**src/app/app.module.ts**
 ```ts
 import { MaterialModule } from '@angular/material';
-// other imports 
+ 
 @NgModule({
+  ...
   imports: [MaterialModule],
   ...
 })
 export class PizzaPartyAppModule { }
 ```
 
-## Include the core and theme styles:
-This is **required** to apply all of the core and theme styles to your application. You can either
-use a pre-built theme, or define your own custom theme.
+## Step 3: Include Theming
+
+Including a theme is **required** to apply all of the core and theme styles to your application. 
+
+To get started with a prebuilt theme, include the following in your app's index.html:
+
+```html
+<link href="node_modules/@angular/material/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
+```
+
+For information on other prebuilt themes, or help building your own theme, check out the [theming guide](./theming.md).
 
-:trident:  See the [theming guide](./theming.md) for instructions.
+## Step 4 (Optional): Gesture Support
```

Comment:
I wouldn't explicitly mark this as optional since slider won't work at all without it. It's more "This is required for some components but we're tolerant of it being missing."